src/web/core/src/main/java/org/geoserver/web/data/store/ParamInfo.java

```diff
@@ -105,7 +108,11 @@ public String getTitle() {
     public boolean isPassword() {
         return password;
     }
-    
+
+    public String getLevel() {
```

Comment:
ParamInfo more or less directly mirrors Param / Parameter, and is only used by the UI code.
Javadocs defining the level are accessible from the Parameter javadocs for the equivalent methods.
Ultimately, the information already exists in a more useful location, although we could maybe reference those classes?